### PR TITLE
Document isEmpty function.

### PR DESCRIPTION
**Pull request by sherfert**
Must wait for https://github.com/neo-technology/neo4j/pull/7897 to get merged.

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala [hunk 1]
```diff
@@ -116,6 +118,39 @@ class PredicateFunctionsTest extends DocumentingTest {
...
+        resultTable()
+      }
+      function("isEmpty(string)", "A Boolean.", ("string", "An expression that returns a string."))
+      query(
+        """MATCH (n)
+          |WHERE isEmpty(n.eyes)
+          |RETURN n.age AS age""".stripMargin, ResultAssertions(r => {
+          r.toList should equal(List(Map("age" -> 61)))
+        })) {
+        p("The ages of all nodes with that have the empty string as the `eyes` property.")
```
Comment (tobias-johansson) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala:150]: It only occurred to me now, that `WHERE isEmpty(n.eyes)` will keep rows where `n.eyes = ''` but drop rows where `n.eyes = null` (i.e. unset). That might seem surprising to a user, so perhaps it should be called out here somewhere?

**Review by tobias-johansson — CHANGES_REQUESTED** [review]

**Review by tobias-johansson — APPROVED** [review]

**Comment by sherfert** [comment]
@davidoliverSP2 Looks like there is a test failing because of a product bug. Let's hold off with merging this PR, I'll let you know when we have fixed the bug in the product.

**Comment by davidoliverSP2** [comment]
@sherfert this looks like the dependant PR in neo4j (neo-technology/neo4j#7897) is now merged. Is this one now good to go?

**Comment by sherfert** [comment]
@davidoliverSP2 yes.

**Pull request merged**